wkcuber/downsampling.py

```diff
@@ -447,6 +452,61 @@ def downsample_mag(
 
 
 def downsample_mags(
+    path: str,
+    layer_name: str = None,
+    from_mag: Mag = None,
+    max_mag: Mag = Mag(32),
+    interpolation_mode: str = "default",
+    cube_edge_len: int = DEFAULT_EDGE_LEN,
+    compress: bool = True,
+    args=None,
+    anisotropic: bool = True,
+):
+    scale = getattr(args, "scale", None) if args else None
+    if not layer_name or not from_mag:
+        layer_name = os.path.basename(os.path.dirname(path))
```

Comment:
So, if one of the attributes `layer_name` and `from_mag` was not provided, you infer both? This means, that if I only provide one of these args, the provided one will be overwritten, nevertheless. This is a bit weird. I guess the reasoning behind this is that the user can either specify the dataset path WITH or without layername/mag.

Maybe add an assertion which enforces that either both or none of the parameters (layername and mag) should be provided.